packages/SwingSet/src/types.js

```diff
@@ -17,15 +19,14 @@
  * TODO: metered...
  *
  * See validateManagerOptions() in factory.js
- * @typedef { 'local' | 'nodeWorker' | 'node-subprocess' | 'xs-worker' } ManagerType
```

Comment:
I thought I re-used `ManagerType` somewhere. I wonder how we can get along without it now.

Reply:
It just moved to later in the same file. Although I think it would be happier in its original location.. I'll move it back.

Reply:
fixed in new commit (7d71fc2e)